File: CSES/1-Introductory-Problems/3-Repetitions/main2.c
#include <stdio.h>

int main()
{
    //Variables
    char in[1000001];
    int temp = 1;
    int out = 1;
    int i;

    //Input
    scanf("%s",in);

    //Calculation
    for (i = 1; in[i]; i++)
    {
        if (in[i] == in[i-1])
        {
            temp++;
            if(temp>= out)out = temp;
        }
        else
        {
            temp = 1;
        }

    }

    //Output
    printf("%d",out);

    return 0;
}

File: Quera/1-Programming-basics/easy/014-Diamond-Printing/Main2.java
import java.util.Scanner;

public class Main2 {

    public static void main(String[] args) {

        //Variables
        int n;
        int spaces;

        //Input
        Scanner getScanner = new Scanner(System.in);

        n = getScanner.nextInt();


        //Calculation & Output
        for (int i = -n; i <= n; i++) {
            spaces = Math.abs(i);

            for (int j = 0; j < spaces; j++) {
                System.out.print(" ");
            }
            for (int j = (n-spaces)*2+1; j > 0; j--) {
                System.out.print("*");
            }
            System.out.print("\n");
        }

        getScanner.close();
    }
}
